docs/Release_Guide/release_steps/met/create_release_on_github.rst

```diff
@@ -1,9 +1,14 @@
 Create Release on GitHub
 ------------------------
 
-- Go to Releases on GitHub and click 'Draft a new release'
-- Set the tag based on the version, starting with 'v', i.e. vX.Y.Z-betaN, vX.Y.Z-rcN, or vX.Y.Z
-- Set the branch value to 'develop' for a development release or 'main_vX.Y' for an official release
-- Choose a release name (i.e. |projectRepo|-X.Y.Z, |projectRepo|-X.Y.Z-betaN, or |projectRepo|-X.Y.Z)
-- Add a link to the release notes from the |projectRepo| User's Guide
-- Attach a PDF of the |projectRepo| User's Guide, if available
+* Go to Releases on GitHub and click 'Draft a new release'
+
+* Set the tag based on the version, starting with 'v', i.e. vX.Y.Z-betaN, vX.Y.Z-rcN, or vX.Y.Z
+
+* Set the branch value to 'develop' for a development release or 'main_vX.Y' for a bugfix or official release
+
+* Choose a release name (i.e. |projectRepo|-X.Y.Z, |projectRepo|-X.Y.Z-betaN, or |projectRepo|-X.Y.Z)
```

Comment:
```suggestion
* Choose a release name (i.e. |projectRepo|-X.Y.Z-betaN, |projectRepo|-X.Y.Z-rcN, or |projectRepo|-X.Y.Z)
```
I noticed that the first and last items in "i.e" were the same and was wondering if it should be as I suggest here (betaN, rcN, empty)?